app/controllers/admin/api/registry/policies_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Admin::Api::Registry::PoliciesController < Admin::Api::BaseController
+  clear_respond_to
+  respond_to :json
+
+  representer ::Policy
+
+
+  # swagger
+  ##~ sapi = source2swagger.namespace("Account Management API")
+  ##~ e = sapi.apis.add
+  ##~ e.path = "/admin/api/registry/policies.json"
```

Comment:
So far I've done this:
![image](https://user-images.githubusercontent.com/11318903/52717611-cd2c8d80-2fa1-11e9-8894-d3c48c811bd2.png)
![image](https://user-images.githubusercontent.com/11318903/52717652-e46b7b00-2fa1-11e9-850f-8007008951d2.png)

Request: `curl -v  -X POST "http://provider-admin.example.com.lvh.me:3000/admin/api/registry/policies.json" -d 'access_token=f46d02db1502511135bd11e54fad2505c40877236c24f23369b667a24c3bf7d9&name=example+name&version=example+version&schema=%7Bexample%3A+'Example+without+validating+the+JSON+at+this+point'%7D'`
Response:
```json
{
 "policy": {
  "id": 1,
  "name": "example name",
  "version": "example version",
  "schema": "{example: 'Example without validating the JSON at this point'}",
  "created_at": "2019-02-13T14:11:57Z",
  "updated_at": "2019-02-13T14:11:57Z"
 }
}
```

It feels very strange to have this link as `POST /admin/api/registry/policies` and the index as `GET /admin/api/policies` 🤔